recipes_source/xeon_run_cpu.rst

```diff
@@ -0,0 +1,291 @@
+torch.backends.xeon.run_cpu
+===========================
+
+There are a set of configurations that would influence the performance of PyTorch inference running on Intel(R) Xeon(R) Scalable Processors.
+To get peak performance, the ``torch.backends.xeon.run_cpu`` script is provided that optimizes the configuration of thread and memory management.
+For thread management, the script configures thread affinity and the preload of Intel(R) OMP library.
+For memory management, it configures NUMA binding and preloads optimized memory allocation libraries (e.g. TCMalloc, JeMalloc).
+In addition, the script provides tunable parameters for compute resource allocation in both single instance and multiple instance scenarios,
+helping the users try out an optimal coordination of resource utilization for the specific workloads.
+
```

Comment:
Please add a "What you will learn" section. It can look like this:

```
What you will learn
-------------------

* How to configure thread and memory management to maximize PyTorch inference performance on Intel(R) Xeon(R) processors.
* How to utilize tools like ``numactl``, ``taskset``, and memory allocators such as TCMalloc and JeMalloc for enhanced performance.